index.html

```diff
@@ -0,0 +1,135 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="node_modules/bootstrap/dist/css/bootstrap.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Karla&family=Ms+Madi&display=swap"
+      rel="stylesheet"
+    />
+    <link href="dist/css/index.css" rel="stylesheet" />
+    <title>PlantScape</title>
+  </head>
+  <body class="background-img">
+    <nav class="navbar navbar-expand navbar-light">
+      <div class="container">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <a class="nav-link" href="#">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Profile</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Feed</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+    <p class="fw-bold text-center mt-5 logo-title">PlantScape</p>
+    <div class="container">
+      <div class="row mt-5">
+        <div class="col-md-5 mx-auto text-center bg">
+          <h2 class="card-title text-center mb-5 mt-5">Log in</h2>
+          <form id="loginForm">
+            <div class="mb-4 col-md-9 mx-auto text-start">
+              <label for="email" class="form-label">Email</label>
+              <input
+                type="email"
+                class="form-control"
+                id="email"
+                name="email"
+                required
+                placeholder="example@example.com"
+              />
+              <div class="invalid-feedback">Please enter a valid email.</div>
+            </div>
+            <div class="mb-4 col-md-9 mx-auto text-start">
+              <label for="password" class="form-label">Password</label>
+              <input
+                type="password"
+                class="form-control"
+                id="password"
+                name="password"
+                minlength="8"
+                required
+                placeholder="********"
+              />
+              <div class="invalid-feedback">
+                Password must be at least 8 characters long.
+              </div>
+            </div>
+            <div class="d-grid mx-auto col-md-5">
+              <button type="submit" class="btn btn-primary mt-3 mb-5">
+                Log In
+              </button>
+            </div>
+          </form>
+        </div>
+
+        <div class="col-md-5 mx-auto text-center bg2 mt-md-0 mt-4">
+          <h2 class="card-title text-center mb-5 mt-5">Sign up</h2>
+          <form id="signupForm" class="pb-5">
```

Comment:
It is a bad user experience to have sign-in and signup forms same time on a page. You could remove one of them and have a link that connects to the other.